src/test/java/uk/nhs/cdss/testHelpers/matchers/FhirMatchers.java

```diff
@@ -108,5 +111,17 @@
         && !requestGroup.hasNote()
         && !requestGroup.hasAction(), "valid 1.1.1 request group");
   }
+  public static Matcher<Parameters> isParametersContaining(
+      Matcher<ParametersParameterComponent>... matchers) {
+    return new FunctionMatcher<>(
+        parameters -> Matchers.contains(matchers).matches(parameters.getParameter()),
+        "is a Parameters resource");
+  }
 
+  public static Matcher<ParametersParameterComponent> isParameter(String name, Resource value) {
+    return new FunctionMatcher<>(
+        parameter -> Matchers.is(name).matches(parameter.getName())
```

Comment:
no need to use matchers here:
```suggestion
        parameter -> name.equals(parameter.getName()),
```
Same for value.